### PR TITLE
Add `arithmetic-operators` as a concept for `cars-assemble`

### PR DESCRIPTION
**Pull request by jmrunkle**
Part of #1611

### CODE REVIEW

File: concepts/arithmetic-operators/introduction.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+They modify the value of a variable by increasing (or decreasing) the value by 1.
+For example:
+
+```go
+a := 10
+a++ // same as a += 1, a == 11
+
+b := 10
+b-- // same as b -= 1, b == 9
+```
```
Comment (ErikSchierboom) [inline concepts/arithmetic-operators/introduction.md:50]: The exercise doesn't require the student to know this to solve the exercise. Could you remove (they're great for the `about.md`)?

```suggestion
```

File: exercises/concept/cars-assemble/.docs/introduction.md [hunk 2]
```diff
@@ -15,6 +15,55 @@ For the sake of this exercise you will only be dealing with:
...
+For example:
+
+```go
+a := 10
+a++ // same as a += 1, a == 11
+
+b := 10
+b-- // same as b -= 1, b == 9
+```
+
```
Comment (ErikSchierboom) [inline exercises/concept/cars-assemble/.docs/introduction.md:66]: The exercise doesn't require the student to know this to solve the exercise. Could you remove (they're great for the `about.md`)?

```suggestion
```

**Review by ErikSchierboom — COMMENTED** [review]
Awesome. Some minor nits.

File: concepts/arithmetic-operators/introduction.md (hunk 1 above)
Comment (junedev) [reply concepts/arithmetic-operators/introduction.md:50]: @ErikSchierboom I would recommend keeping a short version (1 example each) in the introduction and then maybe later adapt the exercise so they can we used. If we put it in about the student will not see them by default and they are needed in the following exercises. Also the Exercism docs say it is allowed to include introduction content that is necessary for the concept itself, that's how I would count this info.

Comment (ErikSchierboom) [reply concepts/arithmetic-operators/introduction.md:50]: I'm fine with that, but I would make it a minimal version then. The current version is quite exhaustive (which is fine for the about.md, but less so for the introduction.md).

File: concepts/arithmetic-operators/about.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+d /= 2 // same as d = d / 2 == 4
+
+e := 16
+e %= 2 // same as e = e % 2 == 0
+```
+
+## Increment and Decrement
+
+There are also two special statements: increment (`++`) and decrement (`--`).
+They modify the value of a variable by increasing (or decreasing) the value by 1.
```
[the remark this answers is not in the document]
Comment (junedev) [reply concepts/arithmetic-operators/about.md:41]: Can here to make a comment about this ... but @ErikSchierboom was faster.

[the remark this answers is not in the document]
Comment (jmrunkle) [reply concepts/arithmetic-operators/about.md:41]: That is actually why I specifically used the word "statements" on line 40. I can be more explicit though.

**Comment by jmrunkle** [comment]
> Awesome. Some minor nits.

Addressed the comments (unless I misunderstood a comment). PTAL

File: exercises/concept/cars-assemble/.docs/introduction.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ For the sake of this exercise you will only be dealing with:
 
 Go supports the standard set of arithmetic operators of `+`, `-`, `*`, `/` and `%` (remainder not modulo).
 
+## Arithmetic Operators
```
Comment (junedev) [inline exercises/concept/cars-assemble/.docs/introduction.md:18]: The sentence on line 16 seems obsolete now.

File: exercises/concept/cars-assemble/.docs/introduction.md [hunk 5]
```diff
@@ -15,6 +15,20 @@ For the sake of this exercise you will only be dealing with:
...
+| Operator | Example        |
+| -------- | -------------- |
+| `+`      | `4 + 6 == 10`  |
+| `-`      | `15 - 10 == 5` |
+| `*`      | `2 * 3 == 6`   |
+| `/`      | `13 / 3 == 4`  |
+| `%`      | `13 % 3 == 1`  |
+
+For integer division, the remainder is dropped (eg. `5 / 2 == 2`).
+
```
Comment (junedev) [inline exercises/concept/cars-assemble/.docs/introduction.md:31]: As said before this is the only file the student really sees so in the spirit of "introducing the concept" I would add one sentence like this even if it currently not part of the exercise:
"Go also has shorthand assignment operators for the operators above, e.g. `a += 5` is short for `a = a + 5`, and it support the increment and decrement operators `++` and `--`."

**Review by junedev — APPROVED** [review]
Two minor comments.

**Comment by jmrunkle** [comment]
> Two minor comments.

Done

File: concepts/arithmetic-operators/introduction.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# Introduction to Arithmetic Operators
```
Comment (junedev) [inline concepts/arithmetic-operators/introduction.md:1]: @ErikSchierboom Is it a problem if the h1 header is more than just the word "Introduction"? (same question for about)

Comment (jmrunkle) [reply concepts/arithmetic-operators/introduction.md:1]: NOTE: if this matters, this is not the only one that would have a problem. Also, it is not mentioned in the contributing guides so it really must be in there if this is a thing that matters.

Comment (ErikSchierboom) [reply concepts/arithmetic-operators/introduction.md:1]: @junedev @jmrunkle It doesn't matter. We strip the h1 heading

**Pull request merged**